pyiron_base/jobs/job/util.py

```diff
@@ -295,11 +303,9 @@ def _job_compress(job, files_to_compress=None):
                         tar.add(name)
             for name in files_to_compress:
                 if "tar" not in name:
```

Comment:
Not introduced here, but would be nice to change ` if "tar" not in name:` to `name != _job_compressed_name(job)`.  Otherwise lgtm.

Reply:
Thanks - I updated the pull request